.github/workflows/release-pr.yml

```diff
@@ -3,7 +3,7 @@ name: Release
 on:
   push:
     branches:
-      - main
```

Comment:
Alternatively this could list them both (and be equal across branches), though I guess there's not really much of a point to that in practice.